ape-config.yaml

```diff
@@ -0,0 +1,39 @@
+plugins:
+  - name: solidity
+  - name: ganache
+
+contracts_folder: contracts/contracts
+
+dependencies:
+  - github: OpenZeppelin/openzeppelin-contracts
+    name: OpenZeppelin
+    version: 4.8.0
+  - name: Interfaces
+    local: ./contracts
+    contracts_folder: ./interfaces
+
+default_ecosystem: ethereum
+
+solidity:
+  evm_version: london
+  version: 0.8.13
+  import_remapping:
+    - 'OpenZeppelin/openzeppelin-contracts@4.8.0/contracts=OpenZeppelin/4.8.0'
+    - 'interfaces=Interfaces'
+
+ganache:
+  server:
+    port: 8545
+  miner:
+    gas_price: 0
+  chain:
+    hardfork: istanbul
```

Comment:
Here we are using `istanbul`. Some lines above we have `  evm_version: london`.
And in our docs we have instructions to use ganache with `london`. Which is the correct one?

Reply:
they are different things. brownie only supports istanbul as hardfork or constantinople